2022-05-12 21:34:05,871 [cuckoo.core.scheduler] INFO: Task #14: acquired machine 192.168.56.102 (label=192.168.56.102)
2022-05-12 21:34:05,871 [cuckoo.core.resultserver] DEBUG: Now tracking machine 192.168.56.101 for task #14
2022-05-12 21:34:05,872 [cuckoo.core.plugins] DEBUG: Started auxiliary module: Replay
2022-05-12 21:34:05,882 [cuckoo.auxiliary.sniffer] INFO: Started sniffer with PID 1473 (interface=vboxnet0, host=192.168.56.101)
2022-05-12 21:34:05,883 [cuckoo.core.plugins] DEBUG: Started auxiliary module: Sniffer
2022-05-12 21:34:05,945 [cuckoo.machinery.virtualbox] DEBUG: Starting vm 192.168.56.102
2022-05-12 21:34:06,263 [cuckoo.machinery.virtualbox] DEBUG: Restoring virtual machine 192.168.56.102 to its current snapshot
2022-05-12 21:34:10,629 [cuckoo.core.guest] INFO: Starting analysis #14 on guest (id=192.168.56.102, ip=192.168.56.101)
2022-05-12 21:34:11,632 [cuckoo.core.guest] DEBUG: 192.168.56.102: not ready yet
2022-05-12 21:34:12,634 [cuckoo.core.guest] DEBUG: 192.168.56.102: not ready yet
2022-05-12 21:34:13,637 [cuckoo.core.guest] DEBUG: 192.168.56.102: not ready yet
2022-05-12 21:34:13,699 [cuckoo.core.guest] DEBUG: 192.168.56.102: not ready yet
2022-05-12 21:34:14,726 [cuckoo.core.guest] INFO: Guest is running Cuckoo Agent 0.10 (id=192.168.56.102, ip=192.168.56.101)
2022-05-12 21:34:14,779 [cuckoo.core.guest] DEBUG: Uploading analyzer to guest (id=192.168.56.102, ip=192.168.56.101, monitor=latest, size=3884763)
2022-05-12 21:34:18,456 [cuckoo.core.resultserver] DEBUG: Task #14: live log analysis.log initialized.
2022-05-12 21:34:22,423 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #14 still processing
2022-05-12 21:34:27,489 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #14 still processing
2022-05-12 21:34:32,566 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #14 still processing
2022-05-12 21:34:37,648 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #14 still processing
2022-05-12 21:34:39,766 [cuckoo.core.resultserver] DEBUG: Task #14 is sending a BSON stream
2022-05-12 21:34:41,405 [cuckoo.core.resultserver] DEBUG: Task #14 is sending a BSON stream
2022-05-12 21:34:41,943 [cuckoo.core.resultserver] DEBUG: Task #14: File upload for 'shots/0001.jpg'
2022-05-12 21:34:41,969 [cuckoo.core.resultserver] DEBUG: Task #14 uploaded file length: 47833
2022-05-12 21:34:42,723 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #14 still processing
2022-05-12 21:34:47,805 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #14 still processing
2022-05-12 21:34:52,862 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #14 still processing
2022-05-12 21:34:57,914 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #14 still processing
2022-05-12 21:35:03,003 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #14 still processing
2022-05-12 21:35:08,208 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #14 still processing
2022-05-12 21:35:11,503 [cuckoo.core.resultserver] DEBUG: Task #14: File upload for 'memory/2712-1.dmp'
2022-05-12 21:35:15,162 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #14 still processing
2022-05-12 21:35:18,701 [cuckoo.core.resultserver] DEBUG: Task #14 uploaded file length: 69494336
2022-05-12 21:35:20,246 [cuckoo.core.guest] INFO: 192.168.56.102: analysis completed successfully
2022-05-12 21:35:20,260 [cuckoo.core.plugins] DEBUG: Stopped auxiliary module: Replay
2022-05-12 21:35:20,262 [cuckoo.core.plugins] DEBUG: Stopped auxiliary module: Sniffer
2022-05-12 21:35:20,262 [cuckoo.machinery.virtualbox] DEBUG: Stopping vm 192.168.56.102
2022-05-12 21:35:21,537 [cuckoo.core.resultserver] DEBUG: Stopped tracking machine 192.168.56.101 for task #14
2022-05-12 21:35:21,537 [cuckoo.core.resultserver] DEBUG: Cancel <Context for LOG> for task 14
2022-05-12 21:35:21,543 [cuckoo.core.scheduler] DEBUG: Released database task #14
2022-05-12 21:35:21,580 [cuckoo.core.plugins] DEBUG: Executed processing module "AnalysisInfo" for task #14
2022-05-12 21:35:21,585 [cuckoo.core.plugins] DEBUG: Executed processing module "BehaviorAnalysis" for task #14
2022-05-12 21:35:21,585 [cuckoo.core.plugins] DEBUG: Executed processing module "Dropped" for task #14
2022-05-12 21:35:21,587 [cuckoo.core.plugins] DEBUG: Executed processing module "DroppedBuffer" for task #14
2022-05-12 21:35:22,255 [cuckoo.core.plugins] DEBUG: Executed processing module "MetaInfo" for task #14
2022-05-12 21:35:24,358 [cuckoo.core.plugins] DEBUG: Executed processing module "ProcessMemory" for task #14
2022-05-12 21:35:24,358 [cuckoo.core.plugins] DEBUG: Executed processing module "Procmon" for task #14
2022-05-12 21:35:24,375 [cuckoo.core.plugins] DEBUG: Executed processing module "Screenshots" for task #14
2022-05-12 21:35:24,378 [cuckoo.core.plugins] DEBUG: Executed processing module "Static" for task #14
2022-05-12 21:35:24,387 [cuckoo.core.plugins] DEBUG: Executed processing module "Strings" for task #14
2022-05-12 21:35:24,390 [cuckoo.core.plugins] DEBUG: Executed processing module "TargetInfo" for task #14
2022-05-12 21:35:24,655 [cuckoo.core.plugins] DEBUG: Executed processing module "NetworkAnalysis" for task #14
2022-05-12 21:35:24,655 [cuckoo.core.plugins] DEBUG: Executed processing module "Extracted" for task #14
2022-05-12 21:35:24,656 [cuckoo.core.plugins] DEBUG: Executed processing module "TLSMasterSecrets" for task #14
2022-05-12 21:35:24,659 [cuckoo.core.plugins] DEBUG: Executed processing module "Debug" for task #14
2022-05-12 21:35:24,660 [cuckoo.core.scheduler] ERROR: Failure in AnalysisManager.run
Traceback (most recent call last):
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/cuckoo/core/scheduler.py", line 766, in run
    self.process_results()
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/cuckoo/core/scheduler.py", line 707, in process_results
    RunSignatures(results=results).run()
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/cuckoo/core/plugins.py", line 342, in __init__
    csv_data = pd.read_csv(csv_file, low_memory = False)
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/pandas/io/parsers.py", line 702, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/pandas/io/parsers.py", line 429, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/pandas/io/parsers.py", line 895, in __init__
    self._make_engine(self.engine)
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/pandas/io/parsers.py", line 1122, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/pandas/io/parsers.py", line 1853, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 387, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 705, in pandas._libs.parsers.TextReader._setup_parser_source
IOError: [Errno 2] File \/home\/cuckoo\/ioc_file\/ioc_ip.csv does not exist: '\\/home\\/cuckoo\\/ioc_file\\/ioc_ip.csv'
